tests/test_web.py

```diff
@@ -40,6 +40,7 @@ def test_cog_translate_webZooms():
     """
     runner = CliRunner()
     with runner.isolated_filesystem():
+        web_tms = morecantile.tms.get("WebMercatorQuad")
```

Comment:
Let's add a test to see if the deprecation warning work and that when using `web_optimized=True` sill work when no tms is passed